trunner/target/armv7a9.py

```diff
@@ -71,8 +73,14 @@ def __call__(self):
             except subprocess.TimeoutExpired as e:
                 raise FlashError(msg=str(e), output=e.stdout.decode("ascii") if e.stdout else None) from e
 
-        self.enter_bootloader()
-        self.wait_prompt()
+        try:
+            self.dut.expect_exact("Waiting for input", timeout=0.5)
```

Comment:
In fact you are expecting "Waiting for input" 2 times (it's already done in enter_bootloader)

```suggestion
```